docs/concepts/input/focus.md

```diff
@@ -0,0 +1,149 @@
+default and must be set to use this feature.
+
+| KeyDeviceType | Device                                    |
+|:--------------|:------------------------------------------|
+| Disabled      | Any key device XY navigation is disabled. |
+| Keyboard      | Keyboard arrow keys can be used.          |
+| Gamepad       | Gamepad controller DPad keys can be used. |
+| Remote        | Remote control can be used.               |
+| Enabled       | All devices can be used.                  |
+
+:::warning
+`KeyDeviceType.Gamepad` is not yet supported, so `XYFocus` navigation is currently limited to `Keyboard` (arrow keys)
+and `Remote` (tvOS most commonly).
+:::
+
+### Navigation Strategy
+
+When 2D directional navigation is enabled, a disambiguation strategy is used to select the navigation target.
+
+| XYFocusNavigationStrategy   | Navigation Target                                                             |
+|:----------------------------|:------------------------------------------------------------------------------|
+| Auto                        | Inherits strategy from ancestor. Projection if no ancestor specifies.         |
+| Projection                  | First element encountered when projecting a line in the navigation direction. |
+| NavigationDirectionDistance | Closest element to the axis of the navigation line.                           |
+| RectilinearDistance         | Closest element based on the shortest Manhattan distance.                     |
+
+### Explicit Navigation
+
+`XYFocus` allows each control to specify an explicit navigation target when a direction is pressed via `XYFocus.Up`, 
+`XYFocus.Down`, `XYFocus.Left`, and `XYFocus.Right`. This has priority over any navigation strategy.
+
+:::warning
+Focus engagement is not yet implemented, so combining Directional focus navigation with controls that also handle 
+directional input themselves may have some limitations, especially with visuals.
+:::
+
+### Example
+
+The following demonstrates how to use Directional focus navigation in a `WrapPanel`. It explicitly allows navigation to 
+wrap from the first to the last element and vice-versa. The `Slider` provides an example of mixing navigation with 
+control interaction. Pressing the Enter key while the `Slider` is focused will begin an interaction where the user will 
+modify the `Slider.Value` instead of causing navigation. Pressing Enter a second time will end the interaction.
+
+```xml
+<Window.Styles>
+    <Style Selector=":is(Control).directional-navigation">
+        <Setter Property="XYFocus.NavigationModes" Value="Enabled" />
+        <Setter Property="XYFocus.UpNavigationStrategy" Value="Projection" />
+        <Setter Property="XYFocus.DownNavigationStrategy" Value="Projection" />
+        <Setter Property="XYFocus.LeftNavigationStrategy" Value="Projection" />
+        <Setter Property="XYFocus.RightNavigationStrategy" Value="Projection" />
```

Comment:
These properties are inheritable, so it might make more sense to set it on top level.